src/styles/blocks/card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: var(--color-white);
+  border: 1px solid var(--color-light-gray);
+  border-radius: 5px;
+}
+
+.card__picture {
+  display: block;
+  margin: 32px auto 0;
+  width: 160px;
+  height: 134px;
+}
+
+.card__header {
+  display: block;
+  margin: 40px var(--text-horizontal-margin) 0;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: var(--color-dark-blue);
+}
+
+.card__description {
+  margin: 4px var(--text-horizontal-margin) 0;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: var(--color-middle-gray);
+}
+
+.card__statistic {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 16px var(--text-horizontal-margin) 0;
+}
+
+.card__reviews {
+  margin: 0;
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color-dark-blue);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin: 24px var(--text-horizontal-margin) 0;
+}
+
+.card__price-title {
+  margin: 0;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: var(--color-middle-gray);
+}
+
+.card__price-value {
+  margin: 0;
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: var(--color-dark-blue);
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 16px var(--text-horizontal-margin);
+  width: 166px;
+  height: 40px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: var(--color-white);
+
+  border: 1px solid transparent;
+  background-color: var(--color-aqua-blue);
+  border-radius: 5px;
+}
+
+.card:hover .card__button {
```

Comment:
```suggestion
.card__button:hover {
```

Reply:
The button shouldn't become hovered when you move the cursor over the card, only - over the button